modules/serverless-apiserversource.adoc

```diff
@@ -0,0 +1,231 @@
+// Module included in the following assemblies:
+//
+// serverless/knative_eventing/serverless-kn-source.adoc
+
+[id="serverless-apiserversource_context"]
+= Using the ApiServerSource
+
+ApiServerSource is an event source that can be used to connect an event sink, such as a Knative service, to the Kubernetes API server.
+ApiServerSource watches for Kubernetes events and forwards them to the Knative Eventing broker.
+
+.Prerequisites
+
+* Using ApiServerSource requires you to have installed Knative Serving and Eventing. For more information, see the documentation on xref:../../serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[Installing OpenShift Serverless].
+
+* You must have created the `default` broker in the same namespace as the one defined in the ApiServerSource YAML. For more information, see the documentation on xref:../../serverless/knative_eventing/serverless-using-brokers.adoc#serverless-using-brokers[Using brokers with Knative Eventing].
+
+* For steps that use `kn`, you will need to have the `kn` CLI installed. For more information, see the documentation on xref:../../serverless/installing-kn.adoc#installing-kn[Installing the Knative CLI].
+
+.Procedure
+
+. Create a Service Account for the ApiServerSource.
++
+You can do this by creating a file named `serviceaccount.yaml` and copying the following sample code into it:
++
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: events-sa
+  namespace: default
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: event-watcher
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - events
```

Comment:
I would say this is probably too much information for this section, since the ApiServerSource docs in this case are specific to Eventing, so I would assume that it should only watch event changes? Do we have any use cases using something like `pod` where we might need to inc. an example for that in docs?